cluster/addons/dns/kube2sky/kube2sky.go

```diff
@@ -58,6 +61,9 @@ var (
 	argDomain              = flag.String("domain", "cluster.local", "domain under which to create names")
 	argEtcdMutationTimeout = flag.Duration("etcd-mutation-timeout", 10*time.Second, "crash after retrying etcd mutation for a specified duration")
 	argEtcdServer          = flag.String("etcd-server", "http://127.0.0.1:4001", "URL to etcd server")
+	argEtcdCAFile          = flag.String("etcd-cafile", "", "Certificate Authority file to secure etcd communication")
+	argEtcdCertFile        = flag.String("etcd-certfile", "", "TLS certificate file to secure etcd communication")
+	argEtcdKeyFile         = flag.String("etcd-keyfile", "", "TLS key file to secure etcd communication")
```

Comment:
We recently fixed this in apiserver. https://github.com/kubernetes/kubernetes/blob/3defbc66803360f9d0fc0a530e31ae82122da006/cmd/kube-apiserver/app/options/options.go#L62

You may want to look at that and figure out if you can reuse anything. It'd be best to be as consistent as possible.